src/main/java/org/thoughtslive/jenkins/plugins/jira/api/Fields.java

```diff
@@ -27,7 +27,10 @@
 @Builder
 public class Fields implements Serializable {
 
-  private static final long serialVersionUID = -6758862154743675930L;
+  private static final long serialVersionUID = 9022728696230161151L;
+
+  @JsonProperty("parent")
+  private Parent parent;
```

Comment:
Ah, yes, I missed the BasicIssue class.  Thanks for the suggestion!  Added commit with change.